package.json

```diff
@@ -7,8 +7,12 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.20.0",
+    "bootstrap": "^4.5.2",
+    "install": "^0.13.0",
+    "npm": "^6.14.8",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-icons": "^3.11.0",
```

Comment:
We are not supposed to touch the package.json and lock files unless we have reason to. This is what will make the application to be dependent on the libraries. Touching this one will have consequences and we don't basically want to touch it.